webpack.config.js

```diff
@@ -46,7 +45,7 @@ const config = [
     },
     devtool: (ENV !== 'production') ? 'source-map' : '',
     resolve: resolveJS(ENV, PATHS),
-    externals,
+    // externals: externalJS(ENV, PATHS),
```

Comment:
Commenting this out was required to resolve the 'jQuery is not defined' issue that the webpack loader was having.  This webpack config was originally intended for use within the CMS where jQuery is globally defined.  We're don't having it globally defined in login-forms, instead it's imported via npm